backend/lcfs/web/api/transaction/services.py

```diff
@@ -189,12 +189,7 @@ async def export_transactions(
         results = await self.repo.get_transactions_paginated(
             0,
             None,
-            [
-                or_(
-                    TransactionView.status == "Approved",
-                    TransactionView.status == "Recorded",
-                )
-            ],
+            [TransactionView.status != "Deleted"],
```

Comment:
This looks a bit dangerous, have we confirmed we want to export Rescinded, Draft, Refused, Submitted, etc?